third_party/ZoKrates/zokrates_stdlib/stdlib/EMBED.zok

```diff
@@ -75,3 +75,7 @@ def u16_to_u32(u16 i) -> u32:
 
 def u8_to_u16(u8 i) -> u16:
     return 0u16
+
+// vector functions
+def vector_add<N>(u32[N] a, u32[N] b) -> u32[N]:
+    return [0; N]
```

Comment:
I think only BV are currently supported in the FHE backend, which is why I left it as u32.

Reply:
Still, it might make sense to support all the basic types, no? It wouldn't be much extra work:

- add vadd_field, vadd_u64, vadd_u32, vadd_u16, vadd_u8 stubs in EMBED
- in builtin_call_, extend the current match clause to catch all of these
- extend the function in `zsharp/term.rs` to support both bitvector and field types
- the current constant folder may or may not work, but I bet there's a small edit distance to working if no

Since this is so simple, I don't see a reason to leave it half-done. But I could be missing something!

Reply:
I see, updated!